docs/managing-data-quality/recovering-failed-events/manual/testing/index.md

```diff
@@ -6,107 +6,55 @@ sidebar_position: 20
-
-See the [main script](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/scripts/Recovery.sc) for additional available convenience functions.
+Which will result in:
+```
+OK! Config valid
+```
 
-## A complete recovery
+Or an detailed error message:
 
-As part of the event recovery project there is the ability to run it in `testOnly` mode. It might be worth while to take a look at the documentation on [running the recovery](/docs/managing-data-quality/recovering-failed-events/manual/running/index.md) first, in order to familiarize yourself with the full process of triggering a recovery job.
+```
+ERROR! Invalid config
 
-You can test a complete recovery, starting from bad rows to getting the data enriched by:
+Instance is not valid against its schema:
+* At $.iglu:com.snowplowanalytics.snowplow.badrows/adapter_failures/jsonschema/1-0-*[0].conditions[0].op:
+  - $.iglu:com.snowplowanalytics.snowplow.badrows/adapter_failures/jsonschema/1-0-*[0].conditions[0].op: does not have a value in the enumeration [Test]
+  ```
 
-- Modifying a `bad_rows.json` file which should contain examples of bad rows you want to recover
-- Adding your recovery scenarios to a `recovery_scenarios.json`
-- If your recovery is relying on specific Iglu repositories additionally to Iglu central, you’ll need to specify those repositories in a `resolver.json`
-- If your recovery is relying on specific enrichments, you’ll need to add them to an `enrichments.json`.
+### Test
 
-Once this is all done, you can run `sbt "project core" "testOnly *IntegrationSpec"`. What this process will do is:
+To run a small-scale recovery test outputting both good and bad events into files, execute:
 
-- Run the recovery on the bad rows contained in `bad_rows.json` according to the configuration in `recovery_scenarios.json`
-- Check that these recovered payloads pass enrichments, optionally leveraging the additional Iglu repositories and enrichments
+```
+snowplow-event-recovery run --config $PWD/scenarios.json -o /tmp -i $PWD/input/ --resolver $PWD/resolver.json
+```
 
-### A custom recovery scenario
+Which for successful recovery will print the progress and output summary:
+```
+Config loaded
+Enriching event...
+Enriched event 2dfeb9b7-5a87-4214-8a97-a8b23176856b
+...
+Total Lines: 156, Recovered: 142
+```
 
-If you’ve written an additional recovery scenario you’ll need to add the corresponding unit tests to `RecoverScenarioSpec.scala` and then run `sbt test`.
+By default no enrichments are configured, but local-only (ones that do not depend on upstream resource storage) enrichments can be configured:
 
-## Output
+```
+snowplow-event-recovery run --config $PWD/scenarios.json -o /tmp -i $PWD/input/ --resolver $PWD/resolver.json --enrichments $PWD/enrichments
```

Comment:
I'd mention the output of this is both a `good.txt` and a `bad.txt` file written locally to the `/tmp` directory.